src/applications/check-in/components/layout/tests/Footer.unit.spec.jsx

```diff
@@ -100,5 +85,17 @@ describe('check-in', () => {
       );
       expect(component.getByTestId('travel-help-block')).to.exist;
     });
+    it('Renders HelpBlock for complete page', () => {
```

Comment:
This is describing the complete page but not testing the complete page. I think just the `it` descriptor needs to be updated to describe this test.